test/test_http_client.py

```diff
@@ -77,6 +77,8 @@ def _stop_server(self):
     def _new_client_connection(self, secure, proxy_options=None):
         if secure:
             tls_ctx_opt = TlsContextOptions()
+            if (TlsCipherPref.PQ_TLSv1_0_2021_05.isSupported):
+                tls_ctx_opt.cipher_pref = TlsCipherPref.PQ_TLSv1_0_2021_05
```

Comment:
let's test this explicitly, instead of sneaking it into our vanilla tests

I'd propose doing it something like:

add an extra param like:
`def _new_client_connection(self, secure, proxy_options=None, cipher_pref=TlsCipherPref.DEFAULT):`

add an explicit test (that's skipped if the cipher isn't supported) something like:
```
    @unittest.skipIf(not TlsCipherPref.PQ_TLSv1_0_2021_05.is_supported())
    def test_connect_pq_tlsv1_0_2021_05(self):
        self._test_connect(secure=True, cipher_pref=TlsCipherPref.PQ_TLSv1_0_2021_05)
```